lib/jsdom/living/window/SessionHistory.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+const traverseHistory = require("./navigation.js").traverseHistory;
+
+// https://html.spec.whatwg.org/#session-history
+class SessionHistory {
+  constructor(initialEntry, window) {
+    this._window = window;
+    this._historyTraversalQueue = new Set();
+    this.entries = [initialEntry];
+    this.currentIndex = 0;
```

Comment:
Yeah, no build step please. I kind of thought at least Flow could catch these things itself without requiring it to be integrated?